web/docs/data-model/backends.md

```diff
@@ -221,10 +221,102 @@ wasp db seed devSeedSimple
 
 Check the [API Reference](#cli-commands-for-seeding-the-database) for more details on these commands.
 
+
+## Seeding the Production Database on Fly.io
+
+Let's say we deployed an app on Fly.
+
+Seeding a production database is a two step process:
```

Comment:
I would like to change this to be formatted like a proper list. See in screenshot that it's on the same line.


![Screenshot 2023-11-09 at 13 23 01](https://github.com/wasp-lang/wasp/assets/2223680/1efce207-baed-43db-ace3-882d3d6763d2)